Source/Core/Common/msvc_sd6_compat.h

```diff
@@ -0,0 +1,226 @@
+// Copyright 2013 Dolphin Emulator Project
```

Comment:
not sure it matters? if you look at other PRs people are basically adding files with any value less <= $CURRENT_YEAR